apps/drained/app.ts

```diff
@@ -65,12 +65,13 @@ const draw = () => {
 };
 
 Bangle.setUI({
-  mode: "clock",
+  mode: "custom",
   remove: () => {
     if (nextDraw) clearTimeout(nextDraw);
     nextDraw = undefined;
   },
 });
+Bangle.CLOCK=1;
```

Comment:
I don't know if we really care if Bangle.CLOCK is set to 1 or not? I just added this so it's similar to calling setUI with mode 'clock'.

Reply:
I don't think it can hurt, I suppose widgets won't run, but perhaps in case there's boot code that might check it. We could pass `clock: 1` in the `setUI` arguments object in fact, what do you think?

Reply:
Yes, you're right - it [sets a watch](https://github.com/espruino/Espruino/blob/master/libs/js/banglejs/Bangle_setUI_F18.js#L76-L80) too. Let's go with this PR as it stands in that case